docs/user-guide/khaas-101.md

```diff
@@ -0,0 +1,134 @@
+
+!!! Warning
+    You should change the default password by editing `NOTEBOOK_PASSWORD=<your_password>` in the `docker-compose.yaml`
+
+
+### k8s deployment
+
+To run the KubeHound as a Service on Kubernetes just use the following [helm files](https://github.com/DataDog/KubeHound/tree/main/deployments/k8s):
+
+
+```bash
+cd ./deployments/k8s
+helm install khaas khaas --namespace khaas --create-namespace
+```
+
+If it succeeded you should see the deployment listed:
+
+```bash
+$ helm ls -A
+NAME    NAMESPACE       REVISION        UPDATED                                 STATUS          CHART              APP VERSION
+khaas   khaas           1               2024-07-30 19:04:37.0575 +0200 CEST     deployed        kubehound-0.0.1
+```
+
+!!! Warning
+    This is an example to deploy KubeHound as a Service in k8s cluster, but you will need to adapt it to your own environment.
+
+## KubeHound collector
+
+In order to use `kubehound` with KHaaS, you need to specify the api endpoint you want to use:
+
+* `--khaas-server` from the inline flags (by default `127.0.0.1:9000`)
+
+Since this is not likely to change in your environment, we advise you to use the local config file. By default KubeHound will look for `./kubehound.yaml` or `$HOME/.config/kubehound.yaml`. As example here we set the default endpoint with disabled SSL.
+
+```yaml
+ingestor:
+    api: 
+        endpoint: "127.0.0.1:9000"
+        insecure: true
+```
+
+!!! Note
+    You can use [kubehound-reference.yaml](https://github.com/DataDog/KubeHound/blob/main/configs/etc/kubehound-reference.yaml) as an example which list every options.
+
+### Dump and ingest
+
+In order to use the collector with KHaaS you need to specify the cloud location you want to dump the k8s resources:
+
+* `--bucket` from the inline flags (i.e. `s3://<your_bucket>`). There is no default value for security reason.
+* `--region` from the inline flags (i.e. `us-east-1`) to set the region to retrieve the configuration (only for s3).
+
+!!! Warning
+    The `kubehound` binary needs to have push access to your cloud storage provider.
+
+If you don't want to specify the bucket every time, you can set it up in your local config file.
+(`./kubehound.yaml` or `$HOME/.config/kubehound.yaml`).
+
+```yaml
+ingestor:
+   blob:
+     # (i.e.: s3://<your-bucket>)
+     bucket: ""
+     # (i.e.: us-east-1)
+     region: ""
+```
+
+!!! Note
+    You can use [kubehound-reference.yaml](https://github.com/DataDog/KubeHound/blob/main/configs/etc/kubehound-reference.yaml) as an example which list every options.
+
+Once everything is configured you just run the following, it will:
+
+* **dump the k8s resources** to the cloud storage provider.
+* send a grpc call to **run the ingestion on the KHaaS** grpc endpoint.
+
+```bash
+kubehound dump remote
+```
+
+or with the flags (for AWS s3):
+
+```bash
+kubehound dump remote --khaas-server 127.0.0.1:9000 --insecure --bucket s3://<your_bucket> --region  us-east-1
+```
+
+!!! Note
```

Comment:
see [docs](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts)
```suggestion
> [!NOTE]
```

Reply:
nop, this is for the wiki, it is not the same. Run the `make local-wiki`